### ISSUE TITLE
ts: update OnChange signature

### ISSUE THREAD

**Issue by RyKilleen** [josdejong/svelte-jsoneditor#174]
The OnChange function currently has the below signature, where `OnChangeStatus` is `any`. It should likely be changed as follows:

```diff
- declare type OnChange = ((content: Content, previousContent: Content, OnChangeStatus: any) => void) | null;
+ declare type OnChange = ((content: Content, previousContent: Content, OnChangeStatus: OnChangeStatus) => void) | null;
```

**Comment by josdejong** [comment]
ahh, that explains the earlier confusion!

The cause is an implicit `any`, see https://github.com/josdejong/svelte-jsoneditor/commit/c2d626f7204790cc86954dc535949dad50822cbd. The reason TypeScript didn't complain about this issue is that I haven't yet configured `strict: true` in the TypeScript configuration. Doing that gives a lot of typing issues right now, it requires some time to fix all of those. It is on the to-do list though.

**Comment by josdejong** [comment]
Fixed now in `v0.9.0`

**Comment by RyKilleen** [comment]
Thanks so much!
